src/MessageDialog.Uno.WinUI/MessageDialog.Uno.WinUI.csproj

```diff
@@ -6,6 +6,7 @@
 		<LangVersion>10.0</LangVersion>
 		<!-- Ensures the .xr.xml files are generated in a proper layout folder -->
 		<GenerateLibraryLayout>true</GenerateLibraryLayout>
+		<DefineConstants>$(DefineConstants);IS_WINUI</DefineConstants>
```

Comment:
Is IS_WINUI the correct constant name? It looks weird using the `IS_` prefix. How is it done in other packages?

Reply:
This constant was based on what uno.toolkit.ui uses. I may be wrong, but I think our packages currently don't support WinUI atm

Reply:
I know it's not used in other nventive packages. I meant Uno packages :)

Reply:
seems uno only uses this in the gallery, I'm not sure it's widespread.
https://github.com/unoplatform/uno.toolkit.ui/blob/c8b8cac7aa63c52467bed62055c26e0195036b26/src/library/Uno.Toolkit.Cupertino/Uno.Toolkit.WinUI.Cupertino.csproj#L13

Reply:
As we're creating a custom one, "WINUI" would probably be better